### ISSUE TITLE
Implement jest tests in stapi  (blocked by the migration to postgres)

### ISSUE THREAD

**Issue by thegentlemanphysicist** [bcgov/digital.gov.bc.ca#961]
We attempted to implement unit testing following the strapi doc suggestions howerver it only works for postgres based strapi CMS, not mongo.  Put this implementation of jest testing on hold until the migration to postgres is done.

Follow this tutorial to tackle this.
https://strapi.io/documentation/developer-docs/latest/guides/unit-testing.html
